### ISSUE TITLE
How to mount jetson onto erlerover?

### ISSUE THREAD

**Issue by revilokeb** [Alexey-Kamenev/scripts_and_more#1]
@Alexey-Kamenev how is it possible to physically mount a jetson dev board onto the erlerover? can this be easily be accomplished?

I would also like to add a stereo camera such as e.g. a stereolabs zed. Could the erlerover carry a jetson tx and a stereo cam?

Many thanks!

**Comment by Alexey-Kamenev** [comment]
Hello,
For Jetson I use [Auvidea J-120](https://auvidea.com/j120/) carrier board which fits surprisingly perfectly right below the Erle Brain. Here is how it looks:
![img_1592s](https://user-images.githubusercontent.com/8118464/27293849-72984708-54cc-11e7-8c15-470ad6ea63f3.JPG)
I **think** you can mount the original Jetson dev board on the rover, there should be no problem as long as you make sure the board is attached firmly as the rover's motor is quite fast :) I haven't tried that myself though.
[ConnectTech](http://connecttech.com/product-category/form-factors/nvidia-jetson-tx2-tx1/#carriers) has carrier boards as well but I haven't tried them.

Yes, you can also mount ZED camera (that's what we use as well as econ Tara on our other platform).
You might need to adjust suspension though - it's too soft by default so use the parts that came with your rover to make it stiffer/higher.

Hope this helps, let me know if you have any questions.

**Comment by revilokeb** [comment]
@Alexey-Kamenev cool many thanks for the very helpful information! I am still invetgating a few pieces related to erlerover and will close this issue if nothing further will pop up.

**Comment by CionnRatz** [comment]
Hi! How did you power the Jetson? Through the Erle battery or installed a dedicated battery? Did you also use a protection circuit for the battery to avoid overdischarge?

Thank you :-)

**Comment by Alexey-Kamenev** [comment]
Hello,
I used a [Venom 3S LiPo](https://www.amazon.com/Venom-1400mAh-11-1V-Battery-Universal/dp/B0199R5O2S/) battery to power both the rover and Jetson.  I did not use protection circuit - I just monitored the battery level in GCS. 

On a related note, we've recently open-sourced our system for the drone, you may want to check it out to find more details on Jetson: [NVIDIA redtail](https://github.com/NVIDIA-Jetson/redtail/), check out the wiki in particular.

**Comment by CionnRatz** [comment]
Thanks!

**Comment by kometa-triatlon** [comment]
@Alexey-Kamenev 
Hi, thanks for the hint about J120. I have a question: How to power it up with a power bank? 
Are there adaptors for the 4-pin power input that J120 uses? Could you tell which model of powerbank/wires/adaptors you use?

**Comment by Alexey-Kamenev** [comment]
We have a detailed build instructions on our [redtail project](https://github.com/NVIDIA-Jetson/redtail/wiki/). We successfully used LiPo 2S, 3S and 4S batteries to power J-120 with Jetson. J-120 accepts voltage range 7-17V. However, please do **not** use power supply that comes with Jetson SDK - it outputs 19V so you can fry your J-120.
Hope this helps and feel free to ask more questions on [redtail](https://github.com/NVIDIA-Jetson/redtail/issues).

**Comment by CionnRatz** [comment]
Dear all, thanks for the hint. One more question. Did you also try to battery power the Jetson TX2 development kit? We would like to start experimenting with this on an rover before buying separate TX2s and carrier boards. Thanks!

**Comment by Alexey-Kamenev** [comment]
Yes, you can power the dev board using a battery. We tried LiPo 3S (5100mAh) and it worked fine. I don't remember the exact input voltage range for devkit board so please refer to the documentation for more details.

**Comment by kometa-triatlon** [comment]
@Alexey-Kamenev 
We have a [Turnigy 3S 11.1V LiPo battery ](https://lipolix.de/produkt/turnigy-5000mah-3s-xt60-20c/) which should fit well for the J120 board. It has two connectors: one XT60 for power and 4-pin JST-XH for balance. Can you say whether it is possible to use it with J120? If yes, are there any adaptors we can use? Thanks.

**Comment by Alexey-Kamenev** [comment]
This battery will work just fine. Our drones/rovers use 3S batteries from Turnigy, Venom and 3DR and everything works great.
Unfortunately, there is no connector readily available that would connect XT60 directly to Jetson so you will need to do some very basic soldering (in case you find such connector - please let me know, I will update our wiki). In case of our Iris+ drone, it already has 12V JST RCY connector so all we had to do is to make the connector JST RCY -> 4-pin Molex Picoblade, see [here](https://github.com/NVIDIA-Jetson/redtail/wiki/3DR-Iris-Setup#jetson-power-wiring) for more details. In case of the rover I just soldered JST RCY female connector to XT-60 connector and used the same cable as in Iris+ drone.
